pkg/ride/tree_estimatorV2.go

```diff
@@ -85,23 +86,25 @@ func (s *estimationScopeV2) restore(pv, pf, po int) {
 }
 
 type treeEstimatorV2 struct {
-	tree  *Tree
+	tree  *ast.Tree
 	scope *estimationScopeV2
 }
 
-func newTreeEstimatorV2(tree *Tree) (*treeEstimatorV2, error) {
+func newTreeEstimatorV2(tree *ast.Tree) (*treeEstimatorV2, error) {
 	r := &treeEstimatorV2{tree: tree}
 	switch tree.LibVersion {
-	case 1:
+	case ast.LibV1:
 		r.scope = newEstimationScopeV2(ConstantsV1, CatalogueV2)
-	case 2:
+	case ast.LibV2:
 		r.scope = newEstimationScopeV2(ConstantsV2, CatalogueV2)
-	case 3:
+	case ast.LibV3:
 		r.scope = newEstimationScopeV2(ConstantsV3, CatalogueV3)
-	case 4:
+	case ast.LibV4:
 		r.scope = newEstimationScopeV2(ConstantsV4, CatalogueV4)
-	case 5:
-		r.scope = newEstimationScopeV2(ConstantsV4, CatalogueV5)
+	case ast.LibV5:
+		r.scope = newEstimationScopeV2(ConstantsV5, CatalogueV5)
```

Comment:
Is it the same bug as above or the new one?

Reply:
The same